### ISSUE TITLE
iOS: VoiceOver: Can't focus all items in long dropdown menus

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#8707]
## Environment

Joplin version: bd5e0fd42a8d20ba480f88b0657d7267b2ae9f7a
Platform: iOS
OS specifics: iOS simulator (also physical iOS device)

## Steps to reproduce

1. Enable VoiceOver
   - If running in a simulator on a Mac, VoiceOver can be enabled from system settings and applies to the iOS simulator as well.
3. Launch Joplin
4. Open Joplin's configuration screen
5. Open the language menu
6. Try to focus a language that starts with a letter after "E" in the alphabet

## Describe what you expected to happen

VoiceOver should be able to select all language items that are selectable when not using VoiceOver.
